src/metabase/cmd.clj

```diff
@@ -133,33 +133,17 @@
         (for [[k v] (partition 2 args)]
           [(mbql.u/normalize-token (subs k 2)) v])))
 
-(defn- resolve-enterprise-command [symb]
```

Comment:
This function is renamed and slightly updated below to call the function immediately.